master/buildbot/util/git.py

```diff
@@ -0,0 +1,77 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+from distutils.version import LooseVersion
+
+
+class GitFeatures(object):
```

Comment:
We usually use the Mixin design pattern in order to handle this.
https://stackoverflow.com/questions/533631/what-is-a-mixin-and-why-are-they-useful

I think this would be more coherent to continue use this pattern

Reply:
Thanks for suggestion. Fixed now.